.github/workflows/pull_request.yml

```diff
@@ -101,7 +101,7 @@ jobs:
       # Launch tests
       #----------------------------------------------
       - name: Launch tests
-        run: poetry run pytest --cov-report xml --cov=robotoff tests
+        run: make tests
```

Comment:
This seems to break the github action:

`Service 'workers' failed to build : failed to export image: failed to create image: failed to get layer sha256:b5c7da884f67639fd3dec4ab67e5950371fdfed950da68b3dc0b40f5199d8682: layer does not exist`

[Full log](https://github.com/openfoodfacts/robotoff/runs/4093308717?check_suite_focus=true)

@alexgarel @zond how do we fix this?

Reply:
Can it be temporary network problems? I looks like the docker image repo was broken.

Reply:
Not sure, but it was[ failing yesterday](https://github.com/openfoodfacts/robotoff/runs/4066110863?check_suite_focus=true) too

Reply:
Does it work locally for you? For me they do.

@alexgarel, @ocervell do you know why the tests would fail due to failure to get a docker image - on GitHub but not locally...?

Reply:
Seems like a known bug see https://stackoverflow.com/questions/51115856/docker-failed-to-export-image-failed-to-create-image-failed-to-get-layer#62409523 and https://github.com/moby/moby/issues/37965
Some suggest adding a `RUN true` to Dockerfile after `COPY` or using BUILDKIT (which I would like :-) )

@ocervell what do you think ?

Reply:
Never seen this issue before, we can try using BUILDKIT by setting the env variable `DOCKER_BUILDKIT=1` in the step `env`.

Reply:
Hmm, `DOCKER_BUILDKIT=1` didn't seem to help :)

Reply:
I added the `RUN true`, let see if it fails or not…

Reply:
This does the trick, now let see if it pass or fail for other reasons.